integration_tests/mockApis/prisonApi.ts

```diff
@@ -0,0 +1,76 @@
+import { stubFor } from './wiremock'
+import {
+  PrisonApiAddress,
+  PrisonApiContacts,
+  PrisonApiEmail,
+  PrisonApiTelephone,
+} from '../../server/services/nomisPrisonerService'
+
+const ping = () =>
+  stubFor({
+    request: {
+      method: 'GET',
+      urlPath: '/ping',
+    },
+    response: { status: 200 },
+  })
+
+const stubGetPrisonerContacts = ({ offenderNo, response }: { offenderNo: string; response: PrisonApiContacts }) =>
+  stubFor({
+    request: {
+      method: 'GET',
+      url: `/api/offenders/${offenderNo}/contacts?activeOnly=true`,
+    },
+    response: {
+      status: 200,
+      headers: { 'Content-Type': 'application/json;charset=UTF-8' },
+      jsonBody: response,
+    },
+  })
+
+const stubGetPrisonerAddresses = ({ personId, response }: { personId: number; response: PrisonApiAddress[] }) =>
+  stubFor({
+    request: {
+      method: 'GET',
+      urlPath: `/api/persons/${personId}/addresses`,
+    },
+    response: {
+      status: 200,
+      headers: { 'Content-Type': 'application/json;charset=UTF-8' },
+      jsonBody: response,
+    },
+  })
+
+const stubGetPrisonerPhones = ({ personId, response }: { personId: number; response: PrisonApiTelephone[] }) =>
```

Comment:
seems odd we are having to make calls, does this stubbing imply that the main contacts call isn't retrieving address etc? And we would end up making a lot of calls just to display contacts for a single prisoner?

Reply:
Thats correct!